src/main/java/com/bookbla/americano/domain/postcard/controller/PostcardController.java

```diff
@@ -0,0 +1,38 @@
+package com.bookbla.americano.domain.postcard.controller;
+
+import com.bookbla.americano.base.resolver.LoginUser;
+import com.bookbla.americano.base.resolver.User;
+import com.bookbla.americano.domain.book.service.dto.BookSearchResponses;
+import com.bookbla.americano.domain.postcard.service.PostcardService;
+import com.bookbla.americano.domain.postcard.service.dto.request.SendPostcardRequest;
+import com.bookbla.americano.domain.postcard.service.dto.response.PostcardTypeResponse;
+import com.bookbla.americano.domain.postcard.service.dto.response.SendPostcardResponse;
+import io.swagger.v3.oas.annotations.Parameter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/postcard")
+@RequiredArgsConstructor
+public class PostcardController {
+
+    private final PostcardService postcardService;
+
+    @PostMapping("/send")
+    public ResponseEntity<SendPostcardResponse> sendPostcard(
+            @Parameter(hidden = true) @User LoginUser loginUser,
+            @RequestBody SendPostcardRequest sendPostcardRequest
```

Comment:
SendPostRequest 내부에서 @Size, @NotNull을 이용한 간단한 유효성 검사를 진행해주셨는데요👍
저 어테이션은 @Valid를 붙여 적용할 수 있어요
```suggestion
            @RequestBody @Valid SendPostcardRequest sendPostcardRequest
```